example/image-classification/train_imagenet.py

```diff
@@ -52,7 +52,11 @@
 
 # network
 import importlib
-net = importlib.import_module('symbol_' + args.network).get_symbol(args.num_classes)
+if args.network == 'inception-resnet-v2':
```

Comment:
Are you trying to do global average pooling? Use global_pool=True 
For the other infer_shape, rather than hard code input_data_shape, I think pass number of filter channels is more appropriate, they won't change regardless of input data shape